projects/plugins/jetpack/changelog/fix-hide-filter-widget-on-empty

```diff
@@ -0,0 +1,4 @@
+Significance: minor
+Type: bugfix
+
+Search: fixed "filters" menu is shown on mobile even if there are no filters to display #18989
```

Comment:
```suggestion
Search: hide "filters" menu on mobile if there are no filters to display
```

This is a user-facing changelog so we don't usually mention the Github PR number in it :)